pkg/codegen/internal/test/testdata/external-resource-schema/go/go.mod

```diff
@@ -4,15 +4,11 @@ go 1.14
 
 // TODO need to bump these to latest after XX merges
```

Comment:
Any idea what this TODO is about? Can probably nuke it

Reply:
Nuked.. It was to remind me of deleting these rewrites once some of the initial go PRs merged... 🤦🏾